Notebooks/SentinelUtilities/SentinelAnomalyLookup/anomaly_lookup.py

```diff
@@ -0,0 +1,241 @@
+"""
+Anomaly Lookup:
+This module provides process flow functions for anomaly lookup.  Method - run is the main entry point.
+It has dependency on .NET library: Microsoft.Azure.CIS.Notebooks.AnomalyLookup.
+"""
+
+import clr
+clr.AddReference("System")
+clr.AddReference('Newtonsoft.Json')
+clr.AddReference("Microsoft.Azure.CIS.Notebooks.AnomalyLookup")
+
+import datetime as dt
+import pandas as pd
+from pandas.io.json import json_normalize
+import copy
+import sys
+import json
+import ipywidgets as widgets
+from ipywidgets import Button, GridBox, Layout, ButtonStyle
+from azure.loganalytics import LogAnalyticsDataClient
+from azure.loganalytics.models import QueryBody
+from System import *
```

Comment:
Not considered good practice to import *

Reply:
Will remove it in next iteration